asset/css/crontab.less

```diff
@@ -0,0 +1,129 @@
+// Cron tab expression widget
+
+.crontab-occurrence {
+  line-height: 1.1em;
+  margin: 0 1em 0 14em;
+
+  p {
+    color: @disabled-gray;
+  }
+}
+
+form.icinga-form .control-group.crontab-wrapper, .monthly-annually-selector {
+  margin-bottom: 0;
+}
+
+form.icinga-form .control-group.crontab-wrapper {
+  margin-top: -1em;
+}
+
+form.icinga-form .crontab {
+  flex: 2;
```

Comment:
Why not just `.crontab`?
Where does this become a flex item? Layout CSS should be self-contained and dependent rules should be applied near the same location and if possible even in a combined rule block.

bad:

```css
.foo {
  display: flex;
}

.bar { // Happens to be inside .foo
  flex: 1 1 auto;
}
```

better:

```css
.foo {
  display: flex;
}

.foo > .bar {
  flex: 1 1 auto;
}
```

perfect: (LESS)

```css
.foo {
  display: flex;

  > .bar {
    flex: 1 1 auto;
  }
}
```